src/views/layouts/admin.hbs

```diff
@@ -11,9 +11,12 @@
     {{>admin_sidebar}}
     {{{body}}}
   </main>
+  {{#if admin}}
```

Comment:
but admin.js without if, coz we need it in all the project

Reply:
solve